bootloader/lkenv/lkenv_v2.go

```diff
@@ -101,6 +101,19 @@ type SnapBootSelect_v2_recovery struct {
 	when not defined or empty, default boot.img will be used */
 	Bootimg_file_name [SNAP_FILE_NAME_MAX_LEN]byte
 
+	/** try_recovery_system contains the label of a recovery system to be
```

Comment:
we also should update the C header file, it's a bit unfortunate that when you made this change, we didn't break anything because the C header file wasn't also updated, maybe we can write some kind of minimal cgo spread test that imports the C header file and the Go package and makes sure that everything is in sync

Reply:
I played a bit with this and have some ideas, still sounds like a followup material.

Reply:
yeah sorry I didn't mean you need a test in this PR, if you have a test that's great and certainly followup material